app/validate.py

```diff
@@ -0,0 +1,85 @@
+import re
+
+
+class Validation:
+    '''Class for returning validation results'''
+
+    def __init__(self):
+        self.status = None
+        self.message = ""
+
+
+def validate_register(json):
+    keys = ['email', 'first_name', 'last_name', 'password']
+    validation = Validation()
+    for field in keys:
+        if field not in json.keys():
+            validation.status = False
+            validation.message = "Missing fields in request data. Include: " +\
+                ", ".join(keys)
```

Comment:
Wrong use of `.join(arg)` in line 19. Use `.format()` e.g. `validation.message = "Missing fields in request data. Include: , {}".format(keys)`